### DISCUSSION TITLE
Add hotkey [F2] to delete file dialog

### DISCUSSION THREAD

**Discussion by Migele** [doublecmd/doublecmd#1232] [Suggestions]
Often I unzip files from big archive (in queue) and then delete it archive file, but for delete I must or wait while file unzip or mouse click key "Add to queue" . I suggest adding hotkey "Add to queue [F2]" on delete file dialog. Thank

**Comment by ags1234** [comment]
+1
Unreal Commander has this and it is very useful.

**Comment by zxcat** [comment]
There is `cm_AddToQueue` command, which you can assign a hotkey and even `queueid` parameter to preselect a queue. But for some reason it only exists for Copy/Move Dialog and not for Delete. Using queued delete a lot sometimes, it would be great to have a hotkey for it.
@alexx2000, at first glance, this command seems easy to extend to the Delete dialog, doesn't it?

**Comment by alexx2000** [comment]
Done.

**Reply by zxcat** [reply]
Thank you!

**Reply by Migele** [reply]
Thank! I'm happy!

**Reply by ags1234** [reply]
Thank you!
